front/src/applications/stdcm/components/StdcmForm/StdcmConfig.tsx

```diff
@@ -173,6 +184,22 @@ const StdcmConfig = ({
     }
   }, []);
 
+  useEffect(() => {
+    if (isPathFindingLoading) {
+      setShowMessage(true);
+    }
+
+    const handleAnimationEnd = () => {
+      setShowMessage(false);
+    };
+
+    pathfindingBannerRef.current?.addEventListener('animationend', handleAnimationEnd);
+
+    return () => {
+      pathfindingBannerRef.current?.removeEventListener('animationend', handleAnimationEnd);
```

Comment:
Sorry to change again but it seems `showBanner` never goes back to false (and the hidden banner is never remove from the dom) if we launch 2 pathfinding in a row : launch a PNO>LEW, clean LEW and then launch a PNO>ARR and watch the element not being removed in the inspector)
```suggestion
    document.addEventListener('animationend', handleAnimationEnd);

    return () => {
      document.removeEventListener('animationend', handleAnimationEnd);
```

Reply:
`document` will catch animations from all elements on the page, so I don't think it's a good idea. For instance if a completely unrelated element happens to animate too then we'd mix things up here.

Reply:
Yeah that's why we changed it before, but somehow the `showMessage` state is not reset to true (only the first time) if we keep it with the ref on the banner

Reply:
Have you tried `useLayoutEffect` and putting the ref on the `.pathFinding-status`?

Reply:
Yep, didn't work

Reply:
Fixed